index.html

```diff
@@ -4415,22 +4421,42 @@ <h3>JSON-LD</h3>
 
         <ul>
           <li>
-Use of JSON-LD keywords in the `@context` value that globally modify document
-value processing, such as global settings of `@base`
+In-line declaration of JSON-LD keywords in the `@context` value that globally
+modify document term and value processing, such as setting `@base` or `@vocab`
           </li>
           <li>
-In-line declaration of JSON-LD contexts in the `@context` property.
+Use of JSON-LD contexts that override declarations in previous contexts, such as
+resetting `@vocab`
+          </li>
+          <li>
+In-line declaration of JSON-LD contexts in the `@context` property
           </li>
           <li>
 Use of full URLs for JSON-LD terms and types (e.g.,
 `https://www.w3.org/2018/credentials#VerifiableCredential` or
 `https://vocab.example/myvocab#SomeNewType`) instead of the short forms of
 any such values (e.g., `VerifiableCredential` or `SomeNewType`) that are
-either explicitly defined as JSON-LD `@context` mappings (e.g.,
-`https://www.w3.org/ns/credentials/v2`) or are implicitly defined via the
-`@vocab` feature that applies to all undefined terms.
+explicitly defined as JSON-LD `@context` mappings (e.g.,
+`https://www.w3.org/ns/credentials/v2`)
           </li>
-       </ul>
+        </ul>
+
+        <p>
+While this specification cautions against the use of `@vocab`, there are
+legitimate uses of the feature, such as to ease experimentation, development,
+and localized deployment. If an application developer wants to use `@vocab` in
+production — even though strongly advised against — they are urged to understand that
+any use of `@vocab` will disable reporting of "undefined term" errors, and
+later use(s) will override any previous `@vocab` declaration(s). Different values
+of `@vocab` can change the semantics of the information contained in the document,
+so it is important to understand whether and how these changes will affect the
+application being developed. Application developers MUST understand every
+JSON-LD context used by their application, at least to the extent that it affects
+the semantics of the terms that are used by their application. Applications MAY
+use JSON-LD <a data-cite="JSON-LD11-API#compaction-algorithms">compaction
+algorithms</a> to transform a document that uses an unknown JSON-LD context
+to one that does not, so the new document's terms will match expectations.
```

Comment:
Some additional detail might be needed, with the addition of that sentence. I think this, because I don't know (and I haven't been able to figure it out, after more than a few minutes reading the linked page and others. I had *thought* I had a good grasp of JSON-LD mechanics, but this makes me wonder!) how the suggested transformation would work. Maybe add some more-descriptive, psuedo-algorithmic language here, with fewer low-level details than the linked page?

Reply:
I'm thinking that this statement is going to require it's own section. Both in the Data Integrity spec, and possibly in the VCDM spec. People seem to be requesting it over here (see item 3): https://github.com/w3c/vc-data-integrity/issues/272#issuecomment-2212258255

If we get consensus to do item 3 (above), then I expect we'll dedicate a section to that sentence. So, we can take it out for now, or leave it in and expand upon it later. I'm fine either way.

Reply:
Either will work for me, for now, if a tracking issue is opened before/upon merge of this PR.

Reply:
This is now being tracked in https://github.com/w3c/vc-data-model/issues/1529